### PR TITLE
feat: updated email format for SQL-based email alerts

### PR DESCRIPTION
**Pull request by JasonD28**
### SUMMARY

Added the relevant SQL statement and a link to the relevant slice for SQL-based email alerts.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:
Screen Shot 2020-08-03 at 5 36 24 PM
After:
Screen Shot 2020-08-04 at 12 48 29 PM

### TEST PLAN

- [x] local test
- [ ] dropbox staging
- [ ] dropbox production

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI

### CODE REVIEW

File: superset/tasks/schedules.py [hunk 1]
```diff
@@ -602,10 +604,16 @@ def deliver_alert(alert: Alert, recipients: Optional[str] = None) -> None:
         textwrap.dedent(
             """\
             <h2>Alert: %(label)s</h2>
-            <img src="cid:screenshot" alt="%(label)s" />
+            <p><b>SQL Statement: </b>%(sql)s</p>
+            <p>Click <a href="%(image_url)s">here</a> or the image below
```
Comment (bkyryliuk) [inline superset/tasks/schedules.py:608]: you can probably do even better here: 
https://stackoverflow.com/questions/17245895/how-to-format-text-in-email-when-using-smtp

e.g. 
* highlight sql section & use separate font
* show the result returned
* use red color
* have a link to the alert history

File: superset/tasks/schedules.py [hunk 2]
```diff
@@ -602,10 +604,16 @@ def deliver_alert(alert: Alert, recipients: Optional[str] = None) -> None:
         textwrap.dedent(
             """\
             <h2>Alert: %(label)s</h2>
-            <img src="cid:screenshot" alt="%(label)s" />
+            <p><b>SQL Statement: </b>%(sql)s</p>
+            <p>Click <a href="%(image_url)s">here</a> or the image below 
+            to view the Slice related to this alert.</p>
```
Comment (bkyryliuk) [inline superset/tasks/schedules.py:609]: s/Slice/ chart

**Review by bkyryliuk — CHANGES_REQUESTED** [review]
let's do couple more small improvements to the message

File: superset/tasks/schedules.py (hunk 1 above)
Comment (bkyryliuk) [reply superset/tasks/schedules.py:608]: also template may live separately, e.g. https://github.com/apache/incubator-superset/blob/master/superset/templates/email/role_extended.txt

Comment (JasonD28) [reply superset/tasks/schedules.py]: I'll leave `show the result returned` as a TODO

File: superset/tasks/schedules.py [hunk 3]
```diff
@@ -598,14 +600,11 @@ def deliver_alert(alert: Alert, recipients: Optional[str] = None) -> None:
...
     if img_data:
         images = {"screenshot": img_data}
-    body = __(
-        textwrap.dedent(
-            """\
-            <h2>Alert: %(label)s</h2>
-            <img src="cid:screenshot" alt="%(label)s" />
-        """
-        ),
+    body = render_template(
```
Comment (bkyryliuk) [inline superset/tasks/schedules.py:603]: add a todo to show results

File: superset/templates/email/alert.txt [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+-->
+<h2 style="color: red;">Alert: {{label}} &#x26A0;</h2>
+<p><b>SQL Statement: </b>
+<code><mark style="background-color: LightGrey; font-size: 1.1em">{{sql}}</mark></code></p>
+<p><a href="{{alert_url}}">View Alert Details</a></p>
```
Comment (bkyryliuk) [inline superset/templates/email/alert.txt:22]: let's have sql in a new line e.g. for the sql queries that are larger

**Review by bkyryliuk — APPROVED** [review]
LG%nits

**Pull request merged**